packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -19,6 +20,10 @@ export type MultiSelectProps = {
   onSelectedValuesChange: MultiSelectOptionsProps["onChange"]
   isOpen: boolean
   onOpenChange: (isOpen: boolean) => void
+  /** A status to convey `error` or `caution` states for the input  */
+  status?: FieldMessageProps["status"]
+  /** A message to provide context for the change in `status`, most commonly validation.  */
+  validationMessage?: FieldMessageProps["message"]
```

Comment:
Noting that we are only really using `error` and `caution` from this. Is it worth just typing those two or leaving as is? Neither our `default` or `success` state impact the UI?

Reply:
You could use `Pick`? Probably better to only include the ones necessary, otherwise it will look like a bug 😬

Reply:
Ah I think this has to be `Extract` since `FieldMessageStatus` is a union of string literals